inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/chart/ChartPanel.java

```diff
@@ -0,0 +1,189 @@
+import org.apache.wicket.markup.head.JavaScriptContentHeaderItem;
+import org.apache.wicket.markup.head.JavaScriptHeaderItem;
+import org.apache.wicket.markup.head.JavaScriptReferenceHeaderItem;
+import org.apache.wicket.markup.html.WebMarkupContainer;
+import org.apache.wicket.markup.html.panel.Panel;
+import org.apache.wicket.model.LoadableDetachableModel;
+import org.apache.wicket.request.cycle.RequestCycle;
+import org.apache.wicket.request.handler.TextRequestHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import de.agilecoders.wicket.webjars.request.resource.WebjarsCssResourceReference;
+import de.agilecoders.wicket.webjars.request.resource.WebjarsJavaScriptResourceReference;
+import de.tudarmstadt.ukp.inception.recommendation.chart.resources.ChartJsReference;
+import de.tudarmstadt.ukp.inception.recommendation.model.LearningCurve;
+
+public class ChartPanel
+    extends Panel
+{
+    private static final long serialVersionUID = -3849226240909011148L;
+
+    private static final Logger LOG = LoggerFactory.getLogger(ChartPanel.class);
+
+    private static final String MID_CHART_CONTAINER = "chart";
+
+    private LoadableDetachableModel<LearningCurve> model;
+    private final WebMarkupContainer chart;
+    private final ChartAjaxBejavior chartAjaxBejavior;
+
+    public ChartPanel(String aId, LoadableDetachableModel<LearningCurve> aModel)
+    {
+        super(aId, aModel);
+        model = aModel;
+        
+        chart = new WebMarkupContainer(MID_CHART_CONTAINER);
+        chart.setMarkupId("canvas");
+        chart.setOutputMarkupId(true);
+        add(chart);
+
+        chartAjaxBejavior = new ChartAjaxBejavior();
+        add(chartAjaxBejavior);
+    }
+    
+    @Override
+    public void renderHead(IHeaderResponse aResponse)
+    {
+        super.renderHead(aResponse);
+        
+        // import Js
+        aResponse.render(JavaScriptHeaderItem
+                .forReference(new WebjarsJavaScriptResourceReference("c3/current/c3.js")));
+        aResponse.render(JavaScriptHeaderItem
+                .forReference(new WebjarsJavaScriptResourceReference("d3js/current/d3.js")));
+
+        // import Css
+        aResponse.render(
+                CssHeaderItem.forReference(new WebjarsCssResourceReference("c3/current/c3.css")));
+
+        aResponse.render(JavaScriptReferenceHeaderItem.forReference(
+                getApplication().getJavaScriptLibrarySettings().getJQueryReference()));
+        
+        aResponse.render(JavaScriptHeaderItem.forReference(ChartJsReference.get()));
+
+        String chartTriggerJavascript = String.join("\n",
+                "$(document).ready(function(){", 
+                "   $.ajax({",
+                "       url:'" + chartAjaxBejavior.getCallbackUrl().toString() + "',",
+                "       type:'post',",
+                "       cache:!1,",
```

Comment:
What is `!1`?

Reply:
It is supposed to be cache:1. Can I change it in the PR I am working on now?

Reply:
I'd suggest to create another PR on #808 to fix just this and keep it separate from whatever PR you are now working on.

Reply:
Should I restore feature/808-project-settings-panel-for-simulated-evalution  branch that you deleted or create a new with the same name?

Reply:
I usually create a new one. I haven't tried if the GitHub UI allows to merge a branch a second time if it receives updates beyond the original merge.